flake-modules/dev/devshell.nix

```diff
@@ -63,6 +63,11 @@
                 ${pkgs.lib.getExe pkgs.python3} -m http.server -d "$doc_derivation"/share/doc
               '';
             }
+            {
+              name = "list-plugins";
+              command = "${pkgs.python3.interpreter} ${./list-plugins.py}";
+              help = "List plugins and get implementation infos";
```

Comment:
The help text could try to imply this prints mostly maintenance related info?

Reply:
I thought so and remembered that the devshell was actually made for... well.. development. So I felt that it was not needed.

If you have a better phrasing, I will be happy to use it though !

Reply:
I think this is fine :+1: